src/inverseRobot.js

```diff
@@ -7,7 +7,22 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedCopy = {};
+  const robotValues = Object.values(robot);
+
+  for (let i = 0; i < robotValues.length; i++) {
+    for (let j = i + 1; j < robotValues.length; j++) {
+      if (robotValues[i] === robotValues[j]) {
+        return null;
+      }
+    }
+  }
```

Comment:
have you checked for absent property here? It's a lot of code for it.  Just use  - .hasOwnProperty() and it will take you two lines of code/

Reply:
agree. thank you